docs/markdown/Getting Help/getting-help.md

```diff
@@ -8,12 +8,21 @@ updatedAt: "2022-02-03T23:16:37.806Z"
 ---
 The [Pants community](doc:the-pants-community) is friendly, welcoming, and passionate about improving the craft of software engineering. If you need help with anything Pants-related, you've come to the right place.  We'd love to hear from you!
 
-Slack
+Community Chat
 -----
 
-The best place to get help is the Pants project's Slack workspace. You can join using [this link](https://docs.google.com/forms/d/e/1FAIpQLSf9zgf-MXRnVDJbrVEST3urqneq7LCcy0zw8qU-GH4hPMn52A/viewform?usp=sf_link).
+The best place to become acquainted with Pants community members and exchange informal chat is at the project's Slack. You can join using [this link](https://docs.google.com/forms/d/e/1FAIpQLSf9zgf-MXRnVDJbrVEST3urqneq7LCcy0zw8qU-GH4hPMn52A/viewform?usp=sf_link). *We use Slack's basic tier, which expires messages after a couple months, so usage is best suited to the more ephemeral or informal conversations that are unlikely to have long-term archival value.*
+
+We encourage you to introduce yourself in the `#welcome` channel!
+
+
+Discussions
```

Comment:
Should we switch the order of these so the one that comes first is "Discussions"?

Reply:
Good idea. Yes.